WBooks/Repository/BookRepository.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BookRepository.swift
+//  WBooks
+//
+//  Created by ignacio.mendez on 09/08/2021.
+//
+
+import Foundation
+import Alamofire
+
+public class BookRepository {
+    
+    // fetch with alamofire request
+    public func fetchBooks(onSuccess: @escaping ([Book]) -> Void, onError: @escaping (Error) -> Void) {
+        // build URL handle response
+        let url = URL(string: "https://ios-training-backend.herokuapp.com/api/v1/books")!
```

Comment:
Let's extract this to a static constant outside the method. We can also divide it in two parts:
´´´Swift
private static let baseURL: String = "https://ios-training-backend.herokuapp.com/api/v1"
private static let booksURI: String = "/books"
´´´